README.md

```diff
@@ -53,9 +53,18 @@ php artisan budget:update
 
 ## Docker
 
-You can run Budget using Docker. Spinning it up (`docker-compose up`) will create 3 containers–for MySQL, PHP and NGINX.
-
-*Disclaimer–currently the Docker setup is missing cronjobs and the queue worker.*
+You can run Budget using Docker Compose. Docker starts 2 containers - one for the database and the other for the application itself.
+
+1. Clone the repository.
+2. Copy the `.env.example` to `.env`, make sure to add an `APP_KEY` and review/replace any other value as you wish.\
+   The `APP_KEY` can be any random value but should be rotated carefully as session cookies are encrypted with this value.
```

Comment:
Is this truly the case? There are no length or format requirements for `APP_KEY`?

Reply:
Good catch. I dug a little deeper: There is no format requirement but the length is important.\
For the cipher in our config `AES-256-CBC` this has to be 32 chars long.\
I will change the phrasing accordingly

Reply:
Cool, thanks.